_events/2023-0510-devoxxuk.markdown

```diff
@@ -0,0 +1,20 @@
+---
+
+eventdate: 2023-05-10 - 2023-05-12
+tz: UTC
+title: Using Apache Kafka and OpenSearch to explore Mastodon
+online: false
+signup:
+    url: https://www.devoxx.co.uk/
+    title: Join Devoxx UK 2023 conference in London
+
+---
+
+Join Olena Kutsenko at DevoxxUK for the session "Using Apache Kafka and OpenSearch to explore Mastodon"
```

Comment:
Would suggest adding your company/title after your name. Also, consider linking to your Twitter/GitHub/or event profile so the audience knows 😃

looks like this is a link for your profile/talk - maybe add that in the post somewhere as well: https://www.devoxx.co.uk/speaker-details/?id=7861

Reply:
Good idea! Added this